actionpack/lib/action_dispatch/cookie_security.rb

```diff
@@ -0,0 +1,68 @@
+module ActionDispatch
+  module CookieSecurity
```

Comment:
I'm not a fan of this top level constant. This class seems to be private API so maybe move to inside the `actionpack/lib/action_dispatch/middleware/cookies.rb ` file?